### PR TITLE
Add CCPP rrtmgp fix

### PR DESCRIPTION
**Pull request by junwang-noaa**
## Description

Point to CCPP branch with RRTMGP lower temperature limitation update.

## Testing

RT has done on hera/orion, will be run on wcoss, jet, gaea and cheyenne.

## Dependencies

CCPP [PR#548](https://github.com/NCAR/ccpp-physics/pull/548)
fv3atm [PR#237](https://github.com/NOAA-EMC/fv3atm/pull/237)
CMEPS [PR#31](https://github.com/NOAA-EMC/CMEPS/pull/31)
ufs-weather-model[ PR#378](https://github.com/ufs-community/ufs-weather-model/pull/378)

### CODE REVIEW

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by climbfuji — COMMENTED** [review]
Looks good. Will approve after ccpp-physics is merged and the submodule is updated.

**Review by climbfuji — APPROVED** [review]

**Pull request merged**
